### DISCUSSION TITLE
Please send help / My zeek is broke :(

### DISCUSSION THREAD

**Discussion by CyberTimeBaby** [Security-Onion-Solutions/securityonion#8686] [Unsupported Versions]
Hello all, I have been having a problem with Zeek Logs not populating in Kibana. It shows as ok when I `sudo so-status` and the output for `sudo docker logs so-zeek` seems healthy, it produced the following 
-`sudo docker logs so-zeek checking configurations ... installing ... creating policy directories ... installing site policies ... generating cluster-layout.zeek ... generating local-networks.zeek ... generating zeekctl-config.zeek ... generating zeekctl-config.sh ... stopping ... stopping worker ... stopping proxy ... stopping manager ... stopping logger ... starting ... starting logger ... starting manager ... starting proxy ... starting worker ...`
I am also still seeing incoming traffic when I tcpdump on the monitor interface as well as the bond0 interface.

I have  checked that my hard drive isn't full(It has several terabytes free) and I am running SO as a bare metal standalone install.
I am running out of ideas here folks

**Comment by reyesj2** [comment]
Do you have zeek logs in **/nsm/zeek/logs**? There should be multiple folders corresponding to the days.

**Reply by CyberTimeBaby** [reply]
Yes I do but they randomly stopped about 4 days ago and none have populated since.

**Reply by reyesj2** [reply]
Have you restarted zeek? Any errors during / following the restart?

```
sudo so-zeek-restart
```
```
sudo docker logs so-zeek
```

**Reply by CyberTimeBaby** [reply]
Yes I have, and it produces no errors upon restart, thank you for the suggestion though.

**Comment by dougburks** [comment]
Are you consuming traffic from a tap or span port?

Restart Zeek and then check `/nsm/zeek/logs/current/` for `stderr.log`, `stdout.log`, and any other clues.

**Reply by CyberTimeBaby** [reply]
So upon restarting I do find an entry in the stderr.log and it reads `1 /opt/zeek/share/zeekctl/scripts/run-zeek/: line 61: ulimit: core file size: cannot modify limit: Operation not permitted` . Also I am using a tap for this traffic.

**Reply by CyberTimeBaby** [reply]
It should be noted that logs are populating in `/nsm/zeek/logs/current/conn.log` but not in Kibana or for previous days.

**Reply by xfaith** [reply]
Do you have any custom scripts for Zeek?  OTX ingest.  A while back someone had the same error line 61.  And it turned out to be OTX.   

https://github.com/Security-Onion-Solutions/securityonion/discussions/6587

**Reply by CyberTimeBaby** [reply]
Nope, running vanilla SO with no custom scripts for anything.

**Reply by dougburks** [reply]
> It should be noted that logs are populating in /nsm/zeek/logs/current/conn.log but not in Kibana or for previous days.

What kind of connections are shown in conn.log?

Is it all of the connections that you expect to see?

If you don't see those connections in Kibana, have you tried our Hunt or Dashboards interface?

Have you checked other logs in /opt/so/log/ for additional clues?

You might want to do another installation (perhaps just in a VM) and then compare the working system to your non-working system side-by-side looking for any differences.
